### DISCUSSION TITLE
如何延时点击？

### DISCUSSION THREAD

**Discussion by wyfang** [gkd-kit/gkd#379] [Q&A]
例如“Appshare”的广告，如果为了符合规则增加积分，需要在广告右上角跳过按钮小于4秒的时候点击才有积分获取。为此，是否有办法获取到秒数？或者就是单纯检测到跳过后等待3000ms再点？

{"id":"info.muge.appshare","name":"AppShare","groups":[{"name":"开屏广告","key":0,"activityIds":["info.muge.appshare.view.launch.LaunchActivity","com.miui.home.launcher.Launcher"],"rules":[{"matches":["[id=\"com.byted.pangle:id/tt_splash_skip_btn\"]"],"snapshotUrls":["https://i.gkd.li/import/12683145","https://i.gkd.li/import/12683173"]},{"matches":["[id=\"info.muge.appshare:id/tv_jump\"][text^=\"跳过\"]"],"snapshotUrls":["https://i.gkd.li/import/12683168"]},{"matches":["ImageView -(4) LinearLayout[childCount=2] > [id=null][text^=\"跳过\"]"],"snapshotUrls":["https://i.gkd.li/import/12683211","https://i.gkd.li/import/12748893"]}]}]}

**Comment by lisonge** [comment]
https://github.com/gkd-kit/subscription/blob/main/src/types.ts
